packages/wonder-blocks-modal/components/modal-launcher/modal-launcher-portal.js

```diff
@@ -0,0 +1,80 @@
+// @flow
+/**
```

Comment:
Are we expecting users to ever use this component? It seems like we should make that explicit here in the comments.

Reply:
Hmm nod, I wonder if maybe it should even be hidden from Styleguidist somehow? I don't think I even want to mention this component to consumers. Could `components/private/modal-launcher-portal.js` make sense?

Reply:
You can explicitly hide it from styleguidist! I think this is the right way: https://react-styleguidist.js.org/docs/cookbook.html#how-to-exclude-some-components-from-style-guide

Reply:
Hmm nod, I wonder if maybe it should even be hidden from Styleguidist somehow? I don't think I even want to mention this component to consumers. Could `components/private/modal-launcher-portal.js` make sense?

Reply:
Hm okay! It feels weird to do that at a global level, but I suppose that's not so bad since this will probably be uncommon.

Reply:
Oh - you don't have to do it on the global level, you can include the ignore option at the section level, too: https://react-styleguidist.js.org/docs/components.html#sections

Reply:
Oh neat, thanks!

Reply:
Nice!